sympy/physics/vector/dyadic.py

```diff
@@ -1,4 +1,8 @@
-from sympy import sympify, Add, ImmutableMatrix as Matrix
+from sympy.core.backend import sympify, AddClass as Add, USE_SYMENGINE
+if USE_SYMENGINE:
+    from sympy.core.backend import Matrix
+else:
+    from sympy import ImmutableMatrix as Matrix
```

Comment:
Is `sympy.ImmutableMatrix` faster than `sympy.Matrix`? While in symengine we only have (mutable) Matrix, so that's what we need to use here. And that's why you cannot just use `from sympy.core.backend import Matrix`, but have to add the switch here to get a faster code if `USE_SYMENGINE==False`?